src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -132,6 +132,7 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Test.Utilities2" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.FSharp" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.FSharp.UnitTests" />
+    <InternalsVisibleTo Include="FSharp.Editor.IntegrationTests" Key="$(FSharpKey)" />
```

Comment:
❗ This must run through the external access layer. You can create a new project for F# integration tests, or you can reuse the existing F# external access project.